jac-uk/assessments
# updating packages

**Author checklist**


---

## What's included?
Describe the changes included in this pull request and highlight dependencies with other repos/tickets

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Run `npm install` 
- Run `npm run serve` to make sure the app compiles and runs

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
Include screen grabs, notes etc.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
